pio/src/iSpindel.cpp

```diff
@@ -971,13 +971,23 @@ float calculateTilt()
 bool testAccel()
 {
   uint8_t res = Wire.status();
-  if (res != I2C_OK)
+    if (res != I2C_OK)
     CONSOLELN(String(F("I2C ERROR: ")) + res);
-
-  bool con = accelgyro.testConnection();
+  
+  bool con = false;
+  if (accelgyro.getDeviceID()==0x34)
+  {
+    con = true;
+  }
+  
+  if (accelgyro.getDeviceID()==0x38)
+  {
+    con = true;
+  }
```

Comment:
```suggestion
  auto id = accelgyro.getDeviceID();
  if ( id ==0x34|| id ==0x38 )
    con = true;
```

Reply:
Please add a comment to which ID what device type matches

Reply:
Updated. I like your update to the code